### PR TITLE
Travis: Update for Moodle39

### PR DESCRIPTION
**Pull request by justusdieckmann**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,99 +1,84 @@
...
-  - DB=pgsql MOODLE_BRANCH=MOODLE_38_STABLE
-  - DB=pgsql MOODLE_BRANCH=master
-  - DB=mysqli MOODLE_BRANCH=MOODLE_35_STABLE
-  - DB=mysqli MOODLE_BRANCH=MOODLE_36_STABLE
-  - DB=mysqli MOODLE_BRANCH=MOODLE_37_STABLE
-  - DB=mysqli MOODLE_BRANCH=MOODLE_38_STABLE
-  - DB=mysqli MOODLE_BRANCH=master
+  jobs:
+    - DB=pgsql MOODLE_BRANCH=MOODLE_35_STABLE
+    - DB=pgsql MOODLE_BRANCH=MOODLE_37_STABLE
```
Comment (Dagefoerde) [inline .travis.yml:25]: _36_ is missing; is that intentional?

Comment (justusdieckmann) [reply .travis.yml:25]: Yes, the image seems a bit weird, but I think support for 3.6 ends this month: https://docs.moodle.org/dev/Releases#Version_support

Comment (Dagefoerde) [reply .travis.yml:25]: also I noticed that the new node version seems to conflict with Moodle 3.6, so it is probably better this way :roll_eyes:

**Comment by justusdieckmann** [comment]
Ready to merge in my opinion.
A new release is needed because of d902b07.

Changelog:
- Remove uses of deprecated behaviour for PHP 7.4.

**Pull request merged**